content/en/registry/instrumentation-js-fs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: OpenTelemetry `fs` Instrumentation for Node.js
```

Comment:
Maybe remove the markdown syntax here?

```suggestion
title: OpenTelemetry fs Instrumentation for Node.js
```

Reply:
I removed the markdown in the title/description everywhere because it's not parsed in the registry

Reply:
I'm a little confused here. Is the markdown intended to be removed or not? It's not removed in the preview of the site: https://deploy-preview-1930--opentelemetry.netlify.app/registry/?s=fs&component=&language=